website/source/docs/job-specification/spread.html.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: "docs"
+page_title: "spread Stanza - Job Specification"
+sidebar_current: "docs-job-specification-spread"
+description: |-
+  The "spread" stanza allows operators to configure desired target percentages for allocations according to node attributes or metadata.
```

Comment:
The "spread" stanza is used to spread placements across a certain node attributes such as datacenter.

Reply:
I find the configure desired target percentages to be confusing. Target percentages of what?

Reply:
_The "spread" stanza allows operators to configure desired target percentages of allocations according to node attributes or metadata_  Is that better? that should have been *of* not *for*